commons/src/lib.rs

```diff
@@ -91,16 +91,34 @@ pub fn ensure_query_params(
     Ok(())
 }
 
-/// Make sure client requested a valid content type.
-pub fn ensure_content_type(
+/// Make sure the client can accept the provided media type.
+pub fn validate_content_type(
     headers: &HeaderMap,
     content_type: &'static str,
 ) -> Result<(), GraphError> {
-    let content_json = header::HeaderValue::from_static(content_type);
+    if !headers.contains_key(header::ACCEPT) {
```

Comment:
Not necessary - duplicated by line 133 (`get` return an `Option`, `map` would throw `Result` if its `None` and `unwrap_or(false)` would result into "Invalid content type" error)